Package.swift

```diff
@@ -16,6 +16,8 @@ let package = Package(
 			exclude: [
 				"LICENSE",
 				"README.md",
+				"release.sh",
+				"Makefile"
```

Comment:
Nitpicky but I'd prefer this to have a trailing comma
```suggestion
				"Makefile",
```